mungegithub/mungers/e2e/e2e.go

```diff
@@ -49,8 +49,9 @@ type BuildInfo struct {
 // RealE2ETester is the object which will get status from a google bucket
 // information about recent jobs
 type RealE2ETester struct {
-	JobNames           []string
-	WeakStableJobNames []string
+	BlockingJobNames    []string
+	NonBlockingJobNames []string
+	WeakStableJobNames  []string
```

Comment:
Agreed, I should be able to use this without the submit queue.
On Jun 10, 2016 8:39 PM, "Eric Paris" <notifications@github.com> wrote:

In mungegithub/mungers/e2e/e2e.go
<https://github.com/kubernetes/contrib/pull/1193#discussion_r66700157>:

> @@ -49,8 +49,9 @@ type BuildInfo struct {
>  // RealE2ETester is the object which will get status from a google bucket
>  // information about recent jobs
>  type RealE2ETester struct {
> -	JobNames           []string
> -	WeakStableJobNames []string
> +	BlockingJobNames    []string
> +	NonBlockingJobNames []string
> +	WeakStableJobNames  []string

We really need to find a better way to get this info... Exposed by
something/somewhere else...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/kubernetes/contrib/pull/1193/files/5e9be452b0e037ef10404eef80538652d53dfd15#r66700157>,
or mute the thread
<https://github.com/notifications/unsubscribe/AAngli_Nc0f5F99B2OmwtQXGSDsgYpsoks5qKi36gaJpZM4IzX9w>
.